metadata-ingestion/src/datahub/ingestion/source/feast.py

```diff
@@ -360,6 +371,38 @@ def _get_on_demand_feature_view_workunit(
 
         return MetadataWorkUnit(id=on_demand_feature_view_name, mce=mce)
 
+    def _get_tags_and_owners(self, obj: Union[Entity, FeatureView, FeastField]) -> list:
+        """
+        Extracts tags and owners from the given object and returns a list of aspects.
+        """
+        aspects: List[Union[GlobalTagsClass, OwnershipClass]] = []
+
+        # Extract tags
+        tag_name = obj.tags.get("name") if obj.tags else None
+        if tag_name:
+            tag_association = TagAssociationClass(tag=builder.make_tag_urn(tag_name))
+            global_tags_aspect = GlobalTagsClass(tags=[tag_association])
+            aspects.append(global_tags_aspect)
+
+        # Extract owner
+        owner = getattr(obj, "owner", None)
+        if owner:
+            owner_association = self._create_owner_association(owner)
+            owners_aspect = OwnershipClass(owners=[owner_association])
+            aspects.append(owners_aspect)
+
+        return aspects
+
+    def _create_owner_association(self, owner: str) -> OwnerClass:
+
+        for mapping in self.source_config.owner_mappings:
+            if mapping["feast_owner_name"] == owner:
+                ownership_type_class: OwnershipTypeClass = mapping["ownership_type"]
+                return OwnerClass(
+                    owner=mapping["datahub_owner_urn"],
+                    type=ownership_type_class,
```

Comment:
What if a user does not provide an ownership type? 

Maybe we have some default here (TECHNICAL owner or something)

Reply:
done, thanks